colabs/raytune/W&B + RayTune.ipynb

```diff
@@ -0,0 +1,147 @@
+{
```

Comment:
<p>You should always start with the Weights &amp; Biases logo image:</p><p> </p><p>&lt;img&nbsp;src="https://i.imgur.com/gb6B4ig.png"&nbsp;width="400"&nbsp;alt="Weights&nbsp;&amp;&nbsp;Biases"&nbsp;/&gt;</p><p> </p><p>Followed by an H1 Header (<code>#</code> in a Markdown cell) with a title. Here, W&amp;B ♥ RayTune, or similar.</p><p> </p><p>I would then write a short paragraph about what RayTune is, what we are, and why we work so well together. Maybe crib some of the text from your report: https://www.wandb.com/articles/distributed-hyperparameter-optimization-at-scale</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>